app/cdap/components/Replicator/utilities/index.ts

```diff
@@ -446,7 +446,12 @@ export function identifyReplicatorEntityFromMetadata(metadata: {
  */
 export function parseErrorMessageForTransformations(message: string) {
   const [_, tablePiece, columnPiece, messagePiece] = message.split(' : ');
-  const table = tablePiece.split(' ')[0];
+  // for mssql source, table name will be in format of schema.table
```

Comment:
Is this fix general or only for MSSQL?

Reply:
both cases will return just the tablename, if it doesn't have '.' the split will just return itself. I assume tablename should not contain '.'

Reply:
Yeah split on `.` returnst he orignal string if it doesn't have a `.` .